profile.graphqls

```diff
@@ -84,6 +84,26 @@ type ProfileTask {
     logs: [ProfileTaskLog!]!
 }
 
+# Profile thread stack anayze tree
+type ThreadStackAnalyzation {
+    # stack code signature
+    codeSign: String!
+    # self include children duration(millisecond)
+    duration: Int!
+    # self exclude children duration(millisecond)
+    excludeChildDuration: Int!
+    # total dump count
+    count: Int!
+    # children of this stack code sign
+    childs: [ThreadStackAnalyzation!]
+}
+
+# Profile analyze result
+type ProfileAnalyzation {
```

Comment:
Are you considering this because of multiple roots?

Reply:
Yes, It's work for multiple roots encase.